server/tests/test_bash_ioc.py

```diff
@@ -18,7 +18,7 @@
     encoding="utf-8",
 )
 
-setup_compose = ComposeFixtureFactory(Path("docker") / Path("test-bash-ioc.yml")).return_fixture()
+setup_compose = ComposeFixtureFactory(Path("tests") / Path("docker") / Path("test-bash-ioc.yml")).return_fixture()
```

Comment:
You don't need all these additional `Path`s:
```py
Path("tests") / "docker" / "test-bash-ioc.yml"
```
(this is one of my favourite synctactic sugars from pathlib...)